libs/datasets/timeseries.py

```diff
@@ -1099,7 +1153,9 @@ def process_series(series):
 
     df_copy.loc[to_exclude[to_exclude].index, CommonFields.TEST_POSITIVITY_7D] = np.nan
 
-    return dataclasses.replace(dataset, timeseries=df_copy).append_tag_df(new_tags.as_dataframe())
+    return dataclasses.replace(dataset, timeseries=df_copy, timeseries_bucketed=None).append_tag_df(
```

Comment:
Why does this need `=None`? isn't this removing any existing timeseries_bucketed objects?

Reply:
I guess I'm worried that bucketed data will just get thrown out very quickly

Reply:
The new `__init__` enforces that exactly one of timeseries_bucketed and timeseries is not None. Whatever was passed in, it stores timeseries_bucketed. Without `timeseries_bucketed=None` replace will pass both to `__init__`. Next I'll deprecate and replace use of timeseries.